packages/middleware-log-errors/README.md

```diff
@@ -0,0 +1,97 @@
+
+## @dotcom-reliability-kit/middleware-log-errors
+
+Express middleware to consistently log errors. This module is part of [FT.com Reliability Kit](https://github.com/Financial-Times/dotcom-reliability-kit#readme).
+
+  * [Usage](#usage)
+    * [`createErrorLogger`](#createerrorlogger)
+    * [configuration options](#configuration-options)
+      * [`includeHeaders`](#optionsincludeheaders)
+  * [Contributing](#contributing)
+  * [License](#license)
+
+
+## Usage
+
+Install `@dotcom-reliability-kit/middleware-log-errors` as a dependency:
+
+```bash
+npm install --save @dotcom-reliability-kit/middleware-log-errors
+```
+
+Include in your code:
+
+```js
+import createErrorLogger from '@dotcom-reliability-kit/middleware-log-errors';
+// or
+const createErrorLogger = require('@dotcom-reliability-kit/middleware-log-errors');
+```
+
+### `createErrorLogger`
+
+The `createErrorLogger` function can be used to generate Express middleware which logs errors to the console and Splunk via [n-logger](https://github.com/Financial-Times/n-logger). It must be added to your Express app _after_ all your application routes:
```

Comment:
Should n-logger be a peer dependency of this package?

Reply:
_maybe_ but unsure, I know nearly all our apps install n-logger anyway but I like the idea that you get everything you need when you install it. Gonna look at how Page Kit and Tool Kit do this.

Reply:
I think _maybe_ peer dependencies aren't right for this case. Happy to be disagreed with but basing this on the following:

1. Page Kit and Tool Kit, from what I can see, define peer dependencies for large third-party tools like React or Webkit rather than internal dependencies (or dependencies managed by Customer Products)

2. I don't think this quite aligns with what peer dependencies are for based on the npm documentation:

    > In some cases, you want to express the compatibility of your package with a host tool or library, while not necessarily doing a require of this host. This is usually referred to as a plugin. Notably, your module may be exposing a specific interface, expected and specified by the host documentation.
    
    This middleware isn't an extension or plugin for n-logger, and so although it's unlikely to be installed in an app which doesn't already have n-logger, I think it's sensible to list it as a regular dependency.